### DISCUSSION TITLE
SPARQL Graph Store Protocol

### DISCUSSION THREAD

**Discussion by namedgraph** [oxigraph/oxigraph#848] [General]
Is the [SPARQL 1.1 Graph Store HTTP Protocol](https://www.w3.org/TR/sparql11-http-rdf-update/) supported?

If it would be, I could try oxigraph as a backend for [LinkedDataHub ](https://github.com/AtomGraph/LinkedDataHub):)

**Comment by Tpt** [comment]
Yes! It is exposed under the `/store` path. The spec is implemented except support of `multipart/form-data`.

**Reply by namedgraph** [reply]
OK, thanks :) I glanced at the README and I didn't see it mentioned together with these:
> [SPARQL 1.1 Query](https://www.w3.org/TR/sparql11-query/), [SPARQL 1.1 Update](https://www.w3.org/TR/sparql11-update/), and [SPARQL 1.1 Federated Query](https://www.w3.org/TR/sparql11-federated-query/).

Maybe worth adding it there?

**Reply by Tpt** [reply]
Indeed. It's only listed in the server/cli README because it is not relevant for the Rust/Python/JS libraries
